bin/docker-build-base

```diff
@@ -1,14 +1,15 @@
 #!/bin/sh
 
-# Builds our base runtime docker image.
+# Builds (or pulls) our base runtime docker image.
 
 set -eu
 
 . bin/_docker.sh
 
-tag="${1:-2017-10-30.01}"
+export tag="2017-10-30.01"
```

Comment:
ehhh, do we really need to export this?  If so, we should give it a better name that isn't likely to conflict

Reply:
we don't need to export this, i've removed it. at some point i wasn't passing it into the subshell correctly and used export as a workaround.